src/FormatCommand.cs

```diff
@@ -22,11 +22,11 @@ internal static RootCommand CreateCommandLineOptions()
                     Description = Resources.A_path_to_a_solution_file_a_project_file_or_a_folder_containing_a_solution_or_project_file_If_a_path_is_not_specified_then_the_current_directory_is_used
                 }.LegalFilePathsOnly(),
                 new Option(new[] { "--folder", "-f" }, Resources.Whether_to_treat_the_workspace_argument_as_a_simple_folder_of_files),
-                new Option(new[] { "--fix-style", "-fs" }, Resources.Run_code_style_analyzers_and_apply_fixes)
+                new Option(new[] { "--fix-style" }, Resources.Run_code_style_analyzers_and_apply_fixes)
```

Comment:
Would it make sense if we change it to `-s` and other one `-a`?
nit: readme.md file will need an update. 🙂

Reply:
I agree that we should change all the short-cut args to be single letter. This also ensures that we don't accidentally alias another options. `-s` and `-a` seem reasonable to me as short hand for `--fix-style` and `--fix-analyzers`

Reply:
It might. I think removing them for a time would be a good thing, because, as you point out, if users are using `-fs` or `-fa` then those will be interpreted as `--folder` and one of the fix options.